cisc/cisc.go

```diff
@@ -380,19 +392,18 @@ func scRoster(c *cli.Context) error {
  */
 func dataUpdate(c *cli.Context) error {
 	cfg := loadConfigOrFail(c)
-	id := cfg.findSC(c.Args().First())
+	id, err := cfg.findSC(c.Args().First())
+	if err != nil {
+		return err
+	}
 	if id == nil {
 		scList(c)
 		return errors.New("Please indicate skipchain-id")
 	}
-	if err := id.DataUpdate(); err != nil {
-		return err
-	}
-	if err := id.ProposeUpdate(); err != nil {
```

Comment:
I haven't looked at CISC code in detail, but why are these not needed anymore?

Reply:
Sorry, no idea. This was part of Linus' hacking around on Friday.

Reply:
After a more in-depth look, I think this is OK. `DataUpdate` and `ProposeUpdate` is called in `cfg.findSC`.